Lib/test/test_pdb.py

```diff
@@ -1614,6 +1614,40 @@ def test_pdb_multiline_statement():
     (Pdb) c
     """
 
+def test_pdb_show_attribute_and_item():
+    """Test for multiline statement
+
+    >>> def test_function():
+    ...     n = lambda x: x
+    ...     c = {"a": 1}
+    ...     import pdb; pdb.Pdb(nosigint=True, readrc=False).set_trace()
+    ...     pass
+
+    >>> with PdbTestInput([  # doctest: +NORMALIZE_WHITESPACE
+    ...     'c["a"]',
+    ...     'c.get("a")',
+    ...     'n(1)',
+    ...     'j=1',
+    ...     'j+1',
+    ...     'r"a"',
+    ...     'c'
+    ... ]):
+    ...     test_function()
+    > <doctest test.test_pdb.test_pdb_show_attribute_and_item[0]>(5)test_function()
+    -> pass
+    (Pdb) c["a"]
+    1
+    (Pdb) c.get("a")
+    1
+    (Pdb) n(1)
+    1
+    (Pdb) j=1
+    (Pdb) j+1
+    2
+    (Pdb) r"a"
+    'a'
+    (Pdb) c
+    """
```

Comment:
The test should probable cover full commands too:

```
(Pdb) next(1)
*** Invalid argument: (1)
      Usage: n(ext)
```

Reply:
Added next and list for test cases.